package.json

```diff
@@ -39,7 +39,7 @@
     "get-next-version": "node scripts/get-next-version.js",
     "postinstall": "yarn-deduplicate --strategy=highest && patch-package && yarn gulp postinstall && ./scripts/run-if-not-ci.sh yarn build",
     "jscodeshift": "jscodeshift -t ./node_modules/js-codemod/transforms/arrow-function-arguments.js",
-    "lint": "eslint --ext .js,.jsx,.ts,.tsx,.json .",
+    "lint": "eslint --ext .js,.jsx,.ts,.tsx,.json, . && eslint --ext .js,.jsx,.ts,.tsx,.json,.vue packages/launchpad",
```

Comment:
Had to do this or it lints `npm/vue` which I don't want to fix right now

Reply:
I noticed that this command will skip linting of `npm/vue` but lint-staged will still catch Vue component changes in `npm/vue` and throw errors.
For example, if you change `npm/vue/examples/vue-cli/src/App.vue` and commit the changes eslint will be mad. Since `npm/vue/examples/vue-cli/` is an example and unlikely that anyone will be making changes then I don't think it's a big deal but I thought I'd mention it.

Reply:
Fair. I don't think we will be modifying `npm/vue` in this feature branch, but we should definitely complete the full linting of all code before merging into develop.

Reply:
@ZachJW34 I ended up linting all Vue files.